cobra/util/util.py

```diff
@@ -57,3 +59,38 @@ def __getitem__(self, item):
         except KeyError:
             value = self[item] = type(self)()
             return value
+
+
+def add_exchange(model, metabolite, demand=True, prefix='EX_', bound=1000.0):
```

Comment:
As mentioned by @midnighter exchange reaction can also export metabolites or both (import and export at the same time). I don't see how the function makes that possible. 

I also agree with the standard prefixes of EX_, DM_, etc. I actually have packages that break if you don't use them...